lib/index.js

```diff
@@ -84,6 +84,9 @@ class DataHubApi {
         )
     let response = await fetch(url)
     if (response.status === 403) {
+      if (!token) {
+        throw { name: 'Forbidden', message: `Not allowed`, res: response }
```

Comment:
what happens when this is thrown? does frontend crash or something? or it just returns 404?

Reply:
404

Reply:
https://github.com/datahq/frontend/pull/136/files#diff-bb503f3d9047bcd4a5730559f1e16560R314